apps/cms/prep-database.sh

```diff
@@ -1,13 +1,13 @@
-if [ ! -z $LAGOON ]
-then
+if [ ! -z $LAGOON ]; then
   # Do not touch database on Lagoon
   exit 0
 fi
 
-if ! test -f web/sites/default/files/.sqlite
-then
+if ! test -f web/sites/default/files/.sqlite; then
   pnpm drupal-install
   pnpm export-webforms
-else
-  pnpm drush cr
 fi
+
+# In any case, re-import translation string.
+pnpm import-translations
```

Comment:
Should we import translations during the Lagoon deployments?

Reply:
I think so? Unless i'm misunderstanding something 😬 
When frontend code changes, there might be new translations and we have to import them.

Reply:
I mean, this file isn't executed in Lagoon context. And I haven't seen `import-translations` anywhere else in the codebase. So perhaps we need to add something to Dockerfile?

Reply:
Ah, no. This has already been there before. Its in a post-deploy step: https://github.com/AmazeeLabs/silverback-template/blob/b5c153a384218b611d112b3eef69d93d893b8afb/.lagoon.yml#L22

Reply:
Ah!